examples/gnrc_minimal/Makefile

```diff
@@ -7,10 +7,10 @@ BOARD ?= native
 # This has to be the absolute path to the RIOT base directory:
 RIOTBASE ?= $(CURDIR)/../..
 
-# Comment this out to disable code in RIOT that does safety checking
+# Uncomment this to enable code in RIOT that does safety checking
 # which is not needed in a production environment but helps in the
 # development process:
-DEVELHELP ?= 1
+DEVELHELP ?= 0
```

Comment:
The comment above and this line don't match, the instructions are to uncomment the line, but it is not a comment at the moment.